community/sharing.md

```diff
@@ -0,0 +1,90 @@
+# Sharing your Mods
+Author(s): code3z (@code3 on the forum)
+
+Once you've created a mod that you're proud of, the next step is to share it.
+Sharing will be a big help to other people. You can also get feedback and help
+from the community. But it may also mean you have to respond to bugs in your
+code and help other people use your code.
+
+## Managing and Sharing Mods using Git
+
+You can use hosted Git repos such as GitHub and Gitlab to upload, manage, share,
+and collaborate on mods. See the Managing Mods section of this guide for more
+information on Git.
+
+## Code headers
+
+Putting a comment at the top of your code helps people understand it,
+use it, and remember where it came from. Consider using this template:
+
+```Javascript OR CSS
+/**
+ * <TITLE>
+ * https://forum.vivaldi.net/topic/<topic>
+	- - -   
+ * Description:        <description>
+ *      <If your description is long, put the second part here>
+ * Filename:          <filename>
+ * Platform:          <Linux/Mac/Windows/All>
+ * Version:           Vivaldi version <version>
+ * Author(s):          <Here you can add your forum username, and username for github or any relevant app (also add any other contributors here)>
+**/
+```
+
+### Versioning Your Mods
+
+Instead of tracking each edit to your mod as a separate version, it's
+easier and sometimes more helpful to include the latest version of Vivaldi that
+it was tested in. Along with the last update time, this will let you and others
+know how up-to-date the mod is.
+
+Because both the forum and Git keep track of posting times, it's not necessary
+to include the last update time in the code header (but you can if you want).
+
+If you do want to version your mods, it would look like this:
+```
+ * Version: 2.1 for Vivaldi 3.9.2289.3
+```
+
+If you want to keep it simple and use the last Vivaldi version, it would look
+like this (see the header template above):
+
+```
+ * Version:           Vivaldi version 3.9.2289.3
+```
+
+## Tagging Forum Posts
+
+Tagging your mods helps others find them. Use the forum tagging feature to
+insert the following tags, in addition to any others that make sense:
+
+ 1. [Mod]
```

Comment:
`modding` seems more common than `mod` (if you update this, consider updating the screenshot as well)